src/libLookup/Lookup.cpp

```diff
@@ -1609,75 +1609,71 @@ bool Lookup::ProcessSetDSInfoFromSeed(const bytes& message, unsigned int offset,
     return false;
   }
 
-  else {
-    bool isVerif = true;
+  bool isVerif = true;
```

Comment:
@ShengguangXiao just to check: without the `else`, the remainder of this function will also be executed for non-lookup every time the node receives `SETDSINFOFROMSEED`. Is this OK?

Reply:
Actually there's no logic change here, because in line 1612 `return false` over there.